v/vertx-micrometer-metrics/vertx-micrometer-metrics_ubi8.4.sh

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : vertx-micrometer-metrics
+# Version       : 3.9.7
+# Source repo   : https://github.com/vert-x3/vertx-micrometer-metrics
+# Tested on     : RHEL UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Vikas Gupta <vikas.gupta8@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=vertx-micrometer-metrics
+PACKAGE_VERSION=3.9.7
+PACKAGE_URL=https://github.com/vert-x3/vertx-micrometer-metrics
+
+mkdir -p /home/tester
+export HOME=/home/tester
+
+# ------- Install dependencies ------
+
+yum -y install git wget make maven gcc-c++ openssl-devel automake autoconf libtool
+yum -y install java-1.8.0-openjdk-devel
+
+#yum -y install gcc-c++.ppc64le
+#yum -y install wget
+#yum -y install git
+#yum install -y openssl-devel.ppc64le
+#yum install -y cmake.ppc64le cmake3.ppc64le
+
+dnf install make maven git sudo wget gcc-c++ apr-devel perl openssl-devel automake autoconf libtool -y
+
+cd $HOME
+echo `pwd`
+
+# --------- Installing cmake version 3.21.2 -----------------
+echo "--------- Installing cmake version 3.21.2 -----------------"
+
+wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2.tar.gz
```

Comment:
Is cmake `3.21.2` a specific requirement here? Does `3.20.2` which is readily available via CentOS AppStream repo not useful here?

Reply:
Removed reinstallation of cmake, tested with cmake-3.20.2 and it worked.